tools/cldr-code/src/test/java/org/unicode/cldr/unittest/LikelySubtagsTest.java

```diff
@@ -310,13 +310,17 @@ public void TestForMissingScriptMetadata() {
             int ch = current.charAt(0);
             int script = UScript.getScript(ch);
             String shortName = UScript.getShortName(script);
-            Info i = ScriptMetadata.getInfo(shortName);
-            if (i == null) {
-                errln("Script Metadata is missing: " + shortName);
-                continue;
-            }
-            if (i.likelyLanguage.equals("und") && !exceptions.contains(shortName)) {
-                errln("Script has no likely language: " + shortName);
+            if (shortName.equals("Chis")) {
+                logKnownIssue("ICU-23171", "ICU4J UScript still supports CHISOI");
+            } else {
+                Info i = ScriptMetadata.getInfo(shortName);
+                if (i == null) {
+                    errln("Script Metadata is missing: " + shortName);
+                    continue;
+                }
+                if (i.likelyLanguage.equals("und") && !exceptions.contains(shortName)) {
+                    errln("Script has no likely language: " + shortName);
+                }
```

Comment:
Let's make this easier to revert.
And we should refer to the still open ICU-23038 rather than 23171 which was unnecessary as a separate ticket.
```suggestion
            Info i = ScriptMetadata.getInfo(shortName);
            if (i == null) {
                if (shortName.equals("Chis")) {
                    logKnownIssue("ICU-23038", "ICU4J UScript still supports CHISOI");
                } else {
                    errln("Script Metadata is missing: " + shortName);
                }
                continue;
            }
            if (i.likelyLanguage.equals("und") && !exceptions.contains(shortName)) {
                errln("Script has no likely language: " + shortName);
            }
```